test_sets/External_Consent_Status_ON.txt

```diff
@@ -0,0 +1,33 @@
+./test_scripts/Polices/External_UCS/External_Consent_ON/001_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_on_data_consent_disallowed.lua Check_that_SDL_returns_DISALLOWED_when_disallowed_by_external_consent_entities_off_exists_externalConsentStatus_ON_and_data_consent_disallowed
```

Comment:
@istoimenova 

Req-t: for OFF:
```
In case
SDL Policies database contains "disallowed_by_external_consent_entities_on" param in "functional grouping"
and SDL gets SDL.OnAppPermissionConsent ("externalConsentStatus: OFF")
and registered application has this "functional grouping" assigned,
SDL must
allow this "functional grouping"
process requested RPCs from such "functional groupings" assigned to mobile app 
```

Req-t for ON:
```
In case
SDL Policies database contains "disallowed_by_external_consent_entities_off" param in "functional grouping"
and SDL gets SDL.OnAppPermissionConsent ("externalConsentStatus: ON")
and registered application has this "functional grouping" assigned,
SDL must
allow this "functional grouping"
process requested RPCs from such "functional groupings" assigned to mobile app 
```

The only difference in value for _externalConsentStatus_ = OFF / ON

From test set for OFF:
```
Check_that_SDL_returns_DISALLOWED_when_disallowed_by_external_consent_entities_on_exists_externalConsentStatus_OFF_and_data_consent_disallowed
```

From test set for ON:
```
Check_that_SDL_returns_DISALLOWED_when_disallowed_by_external_consent_entities_off_exists_externalConsentStatus_ON_and_data_consent_disallowed
```

The only difference in value for _externalConsentStatus_ = OFF / ON

Please clarify what is wrong?